tilauskasittely/tilaus_osto.php

```diff
@@ -131,6 +131,35 @@
   enable_ajax();
 }
 
+if ($tee == 'lisaa_aiemmalle_riville') {
```

Comment:
@tiere Tää on nyt vähän vaarallista koodia. Syöttämällä kivat parametrit suoraan urliin niin saa varmaan homman aika sekaisin. PItäis varmaan joka queryssä olla ehtona, että operoidaan vain käyttäjän keskenolevaa tilausta (kukarow[kesken]) ja sen lisäks pitäis tsekata, että päivitettävät rivit ovat ostorivejä ja sellasia ostorivejä joita ei olla vielä viety varastoon.